src/lsp-commands.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2019 Red Hat, Inc. and others.
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *     http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ConfigurationTarget, Disposable, commands, workspace } from 'vscode';
+
+namespace CommandKind {
+  export const COMMAND_REFERENCES = "quarkus.command.references";
+  export const COMMAND_IMPLEMENTATIONS = "quarkus.command.implementations";
+  export const COMMAND_OPEN_URI = "quarkus.command.open.uri";
+  export const COMMAND_CONFIGURATION_UPDATE = "quarkus.command.configuration.update";
+}
+
+export function registerConfigurationUpdateCommand(): Disposable {
+  return commands.registerCommand(CommandKind.COMMAND_CONFIGURATION_UPDATE, (configItemEdit: ConfigurationItemEdit) => {
+    if (configItemEdit.section === 'quarkus.tools.validation.unknown.excluded') {
```

Comment:
do we need this condition?

Reply:
the reason I have this condition is because of this feedback from a while ago: https://github.com/redhat-developer/vscode-quarkus/pull/152#discussion_r345753500

do you think we don't need it?

Reply:
I would prefer remove it to have no dependency to Quarkus. If we find issues, we will fix it by adding a parameter for instance. But please keep this code with  no dependencies to Quarkus.